develop/parachains/intro-polkadot-sdk.md

```diff
@@ -50,6 +50,21 @@ Every blockchain platform relies on a decentralized network of computers—calle
     - Stored as a part of the chain state
     - Also known as State Transition Function (STF)
 
+``` mermaid
+
+graph TB
+    subgraph sg1[Substrate Node]
+        B[Wasm Runtime -</br>application logic]
+        I[RuntimeCall Executor]
+        subgraph sg2[Client]
+            direction TB
+            C[Network and blockchain</br>infrastructure activity]
```

Comment:
This description feels weird. The word `activity` does not sound good to me. Maybe replacing it with `services` makes it work

Reply:
Cool, cool! I just pulled this off substrate docs. Will update!